guides/doc-Upgrading_and_Updating/topics/upgrading_overview.adoc

```diff
@@ -38,6 +38,11 @@ For more information, see {AdministeringDocURL}Backing_Up_Server_and_Proxy_admin
 * Plan for updating any scripts you use that contain {Project} API commands because some API commands differ between versions of {Project}.
 For more information about changes in the API, see the Red Hat Knowledgebase article https://access.redhat.com/articles/4396911[API Changes Between {Project} Versions] on the Red{nbsp}Hat Customer Portal.
 
+ifdef::satellite[]
+NOTE: {Project} {ProjectVersion} is released only on {EL} 8. You must upgrade {Project} {ProjectVersionPrevious} from {EL} 7 to {EL} 8.
+For more information on upgrading from {EL} 7 to {EL} 8, see xref:upgrading-{project-context}-or-proxy-in-place-using-leapp_{context}[].
```

Comment:
Is this actually correct in master? We only have the upgrade docs in 3.3 and 3.4 so shouldn't this link to the `{ProjectVersionPrevious}` upgrade part?

Reply:
@evgeni Can you confirm?